draft-ietf-capport-architecture.xml

```diff
@@ -256,6 +255,11 @@
         portal web application, and (c) legacy devices must continue to be
         supported.
         </t>
+        <t>
+        If User Equipment supports Captive Portal API, it MUST use TLS to communicate with
+        Captive Portal API and MUST validate API server TLS certificate. If certificate 
+        validation fails, it MUST NOT proceed with any of the behavior described above.
```

Comment:
I don't know what to cite here.  But it seems like you might use `<xref target="RFC2818"/>` as a normative reference.

Reply:
https://xml2rfc.tools.ietf.org/public/rfc/bibxml/reference.RFC.2818.xml